Makefile

```diff
@@ -1,7 +1,13 @@
 .PHONY: tests docs
 
 tests:
-	pytest tests
+	pytest tests -sv
+
+integrationtests:
+	pytest tests -sv -m "integration"
+
+unittests:
+	pytest tests -sv -m "not integration"
```

Comment:
I think "unittests" mark would be better instead of integration/not integration?